futures-util/src/future/abortable.rs

```diff
@@ -0,0 +1,131 @@
+use futures_core::future::Future;
+use futures_core::task::{self, AtomicWaker, Poll};
```

Comment:
```
use crate::task::AtomicWaker;
```

Reply:
Yeah, that changed after I wrote this. Will rebase.